src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,56 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerService;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+
 public class Main {
     public static void main(String[] args) {
-        // test your code here
+        final Injector injector = Injector.getInstance("mate.jdbc");
+        final DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+        final ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+        Manufacturer manufacturerA = new Manufacturer(1L, "A", "AA");
```

Comment:
Do not set ids manually, as the database should generate them automatically. Just remove the id parameter from the constructors of Manufacturer and Driver objects.